Uplift/Views/GymDetail/Facilities/MiscellaneousInfoCell.swift

```diff
@@ -54,8 +49,15 @@ class MiscellaneousInfoView: UIView {
             .paragraphStyle: style,
             .foregroundColor: UIColor.primaryBlack
         ]
+        
+        let string = miscInfo.joined(separator: "\n")
 
         return NSAttributedString(string: string, attributes: attributes)
     }
 
+    static func getHeight(for miscInfo: [String], font: UIFont = UIFont._14MontserratRegular!) -> CGFloat {
+        let nsString = generateAttributedString(for: miscInfo, font: font)
+        return nsString.size().height
+    }
```

Comment:
Can we standardize a method of implementing the `getHeight` functions for these cells? 
It's currently this `NSAttributedString.size()` function in this cell, but manually calculated in the other cells.
I'm not familiar with how this `.size()` function works, but if we test on cells with different #s of lines and it looks to be functioning correctly, I think this method would be better than manual calculation, with the downside of making the `generatedAttributedString` functions `static` -- not sure what's preferred in that regard @kevinchan159

Reply:
`NSAttributedString.size()` is only used for calculating a string's height, which works here because the cell's only view is a text view. Other cells have more UI elements and require more complex ways of calculating height.

Reply:
The other views also have text views in them though -- I think we should standardize how we calculate the height of the text views themselves, despite having additional calculations in the `getHeight`

Reply:
commenting to keep track -- to be tackled in another PR